exercises/hello-world-service/hello_world_service.bal

```diff
@@ -2,9 +2,13 @@ import ballerina/http;
 import ballerina/log;
 
 // By default, Ballerina exposes a service via HTTP/1.1.
-service<http:Service> hello bind { port: 9090 } {
+service hello on new http:Listener(9090) {
 
-    // Invoke all resources with arguments of server connector and request.
-    sayHello(endpoint caller, http:Request req) {
+    resource function sayHello(http:Caller caller, http:Request req) {
```

Comment:
Can we add an updated comment here?

Reply:
Fixed with https://github.com/exercism/ballerina/pull/52/commits/fe87502ed19504424c0e5ff654e0c4f3add1132e